docs-markdown/src/controllers/image-controller.ts

```diff
@@ -63,10 +63,6 @@ export function pickImageType() {
                 applyImage();
                 commandOption = "image";
                 break;
-            case "image":
```

Comment:
unreachable code, case statement above is the same, git blame shows https://github.com/microsoft/vscode-docs-authoring/commit/576a297808852c8f152ebac307a9fc86c9a497c9

Reply:
Good call.